developer_docs/ansible_dev_setup.asciidoc

```diff
@@ -67,6 +67,13 @@ Enable feature
 :enabled: true
 ....
 
+Set path to `ansible-runner-environment`
+[source, bash]
+....
+# vi ~/.bash_rc
+export SMART_PROXY_ANSIBLE_ENVIRONMENT_WRAPPER="/path/to/smart_proxy_ansible/bin/ansible-runner-environment.sh"
```

Comment:
This could be made easier with `export SMART_PROXY_ANSIBLE_ENVIRONMENT_WRAPPER="$(bundle exec gem which smart_proxy_ansible | sed 's|/lib/smart_proxy_ansible.rb||')/bin/ansible-runner-environment.sh"`

Reply:
That gives me `Could not locate Gemfile or .bundle/ directory`

Reply:
Assuming you run it from your smart-proxy checkout